content/en/docs/troubleshooting.md

```diff
@@ -112,7 +112,7 @@ Specifically, this error indicates that the artifact is signed by an unknown ide
 
 To confirm the trust store and trust policy are configured correctly:
 
-1. If you have multiple trust policies in `trustpolicy.json` file, confirm you are using the correct trust policy for this artifact.
+1. If you have multiple trust policies in `trustpolicy.json` file, confirm you are using the correct trust policy for this artifact by running `notation verify IMAGE -v`. Enabling the `-v` flag provides you with more information about the trust policy used for verifying the artifact.
```

Comment:
The command `notation verify IMAGE -v` cannot be used to confirm using the correct trust policy. Since it is due to the error from this command, users want to do the troubleshooting.